code/modules/projectiles/projectile/special.dm

```diff
@@ -13,6 +13,21 @@
 		empulse(get_turf(A), 1, 1)
 	..()
 
+/obj/item/projectile/ionsmall
+	name = "ion bolt"
+	icon_state = "ion"
+	damage = 0
+	damage_type = BURN
+	nodamage = 1
+	layer = PROJECTILE_LAYER
+	flag = "energy"
+	fire_sound = 'sound/weapons/ion.ogg'
+
+/obj/item/projectile/ionsmall/Bump(atom/A as mob|obj|turf|area)
+	if(!bumped && ((A != firer) || reflected))
+		empulse(src, 0, 1)
```

Comment:
If I'm not mistaken, this actually *won't* ion the thing you hit. It will ion the tile on which the bullet was when it hit the thing, which is a tile away from what it hit.